src/themes/clean/templates/elements/journal/issue_list.html

```diff
@@ -1,13 +1,13 @@
 <ul>
     {% for issue in issues %}
         <li>
-            <a href="{% url 'journal_issue' issue.id %}" 
-                aria-label='{{ issue.display_title_a11y }} ({{ issue.articles.all.count }} {% trans "items" %})' 
+            <a href="{% url 'journal_issue' issue.id %}"
+                aria-label='{{ issue.display_title_a11y }} ({{ issue.articles.all.count }} {% trans "items" %})'
```

Comment:
Try to avoid formatting changes that are not related to the issues we are addressing.